.travis.yml

```diff
@@ -10,12 +10,14 @@ python:
 install:
   - sudo apt-get install -y - npm libgnutls28-dev
   - sudo npm install -g markdownlint-cli
-  - pip install flake8
+  - pip install flake8 coverage
+  - python setup.py install
```

Comment:
Because this isn't a real test and it does produce quite a number of logs making actual errors in the tests hard to spot. You can still access the logs if you want to and they are prominently displayed if they fail.